engine-tests/src/tests/promise_results_precompile.rs

```diff
@@ -101,7 +101,7 @@ fn test_promise_result_gas_cost() {
 
     assert!(
         utils::within_x_percent(
-            5,
+            10,
```

Comment:
It's `2x`. Any ideas why?

Reply:
Because we've got the [error](https://github.com/aurora-is-near/aurora-engine/actions/runs/11834374670/job/32974926364#step:8:1227) from time to time on the CI. On localhost I've got: 

`base cost: 107 and promise result base cost: 111`

which is within 3.6 % 
But 102 and 111 within 8.1 %.